modules/virt-importing-vm-datavolume.adoc

```diff
@@ -61,12 +60,12 @@ metadata:
   creationTimestamp: null
   labels:
     kubevirt.io/vm: vm-fedora-datavolume
-  name: vm-fedora-datavolume
+  name: <vm-fedora-datavolume>
 spec:
   dataVolumeTemplates:
   - metadata:
       creationTimestamp: null
-      name: fedora-dv
+      name: <fedora-dv>
     spec:
       pvc:
```

Comment:
As I mention in the virt-importing-vm-to-block-pv section, we now recommend the use of 'storage' instead of 'pvc' since in most cases the volumeMode and accessMode options can be omitted and filled in automatically by the system.